lib/src/country_coder_super.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:point_in_polygon/point_in_polygon.dart';
+import 'package:latlong2/latlong.dart';
+
+abstract class CountryCoderSuper{
+
+  List<Map<String, dynamic>>  get countries;
```

Comment:
Should contain a list of class instances. I proposed the `GeoFeature` class. This should then also be returned by `resolveCountry`.

Perhaps `GeoFeature`  should be named differently like `GeoBoundary`, `GeoRegion`, `GeoArea`